### ISSUE TITLE
OpenSSL and Postgres immediately exit after docker-compose up -d

### ISSUE THREAD

**Issue by jaydusk** [cyberark/conjur-quickstart#10]
I had a lab working just a few days ago, was able to perform the entire quickstart process and successfully run program.sh.   As of today, when i execute docker-compose up -d, everything appears to start, but then docker ps -a shows:

openssl and postgres_database as Exited just after everything starts up.  

Running docker-ce Docker version 19.03.5, build 633a0ea838 on Ubuntu 18.04.4

**Comment by uCatu** [comment]
Hi @jaydusk 
Looks like a Duplicate of #11 
Can you confirm?

**Comment by jaydusk** [comment]
Yes #11 is exactly what I'm seeing - the Postgres and openssl exit immediately.

**Comment by uCatu** [comment]
Thank you, if you can supply the log of `postgres` container by running 
`docker logs postgres_database`
before exit, it will assist me to verify the issue.

**Comment by sgnn7** [comment]
@jaydusk I just want to note here that the openssl exit is perfectly fine and expected. The only thing that container does at the moment is to create some certificates and does not need to keep running. Some of this behavior might have changed in the recent commit so it might not be the same as it behaved earlier.

tl;dr: the new openssl behavior is correct but pg is not.

I'll close the other issue and link to this one so that we don't have a duplicate.

**Comment by sgnn7** [comment]
Looks like PG docker container might have changed some required env vars in PG 9.4 and broke things. I might take a look this morning. This was a fix that was done in Conjur: https://github.com/cyberark/conjur/pull/1336.

**Comment by amarrocco84** [comment]
Here's an output of the logs.
![postgreslogs](https://user-images.githubusercontent.com/60936141/74751065-bbc43400-523a-11ea-9b2d-93d0853bf7a3.JPG)

**Comment by sgnn7** [comment]
@amarrocco84 Got a fix PR in #12 - just waiting for someone from our team to review it.

**Comment by sgnn7** [comment]
@amarrocco84 Fix is merged - please let us know if that fixes your problem. I'll leave this open until we hear from you today.

**Comment by amarrocco84** [comment]
Looks stable so far. I will put it through its paces a bit more but at least the conjur_server container isn't stuck in an infinite loop and the postgres_database container is up now.

![cyberarkossworking](https://user-images.githubusercontent.com/60936141/74761463-15802a80-524a-11ea-9241-d38b3fafe87b.JPG)

**Comment by sgnn7** [comment]
@amarrocco84 Thank you for the feedback! I will close this issue for now but feel free to open a new issue if you find that there are further problems :)

**Comment by jaydusk** [comment]
Thank you all @uCatu  @sgnn7  for your assistance with this.  Your prompt response is SO much appreciated and a great example of the excellence of CYBR.

**Comment by santsant93** [comment]
Hey I've the issue where openssl is not working. Whereas all the other pods are up and good. Any idea what needs to be done?
![Screen Shot 2022-10-18 at 2 52 30 AM](https://user-images.githubusercontent.com/116047220/196316821-623a3c95-9a29-482d-bc45-112e65549aa5.png)

**Comment by santsant93** [comment]
@sgnn7

**Comment by sgnn7** [comment]
Hey @santsant93,
Sadly, I no longer work on Conjur :(
It may be good for you to open a new issue for visibility since this one is closed.
